pages/blog/testing-session-storage-selenium-node.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Testing sessionStorage and localStorage with Selenium (Node)"
+date: "2021-02-18"
+author: "Sean C Davis"
+pullQuote: "How to test for key-value pairs in sessionStorage and localStorage using Selenium Node."
+image: testing-session-storage-selenium-node/selenium-session.png
+tags:
+  - engineering
+  - notes
+---
+
+We have [a feature on this site that is using `sessionStorage`](/blog/getting-previous-path-nextjs) to send analytics data we want to capture. It's an important feature and so we looked to do what we _should_ — write test(s) to cover the use case(s).
+
+This website is a [Next.js](https://nextjs.org/) application that uses [Jest](https://jestjs.io/) as our test runner and [Selenium WebDriver](https://www.selenium.dev/) for integration test help.
```

Comment:
Specifically, we use https://github.com/alexeyraspopov/jest-webdriver and https://github.com/alexeyraspopov/jest-webdriver/tree/master/packages/jest-environment-webdriver, which is the package choosing to call "driver" "browser"